comment/manager/comments.py

```diff
@@ -1,8 +1,15 @@
+from django.conf import settings
 from django.contrib.contenttypes.models import ContentType
 from django.db import models
 
+ALLOWED_FLAGS = getattr(settings, 'COMMENT_FLAGS_ALLOWED', 10)
 
 class CommentManager(models.Manager):
+    def get_queryset(self):
```

Comment:
Overriding this method is no longer necessary if we change the field as suggested in the `Flag` model.

Reply:
This method was trying to filter the comments that have been flagged beyond a certain count. How is this concerned with any of such suggested change?

Reply:
I don't see any reason to override this method and this actually will lead to the following problem:
```python
Comment.objects.all()  # => the result is all comments are FLAGGED
```
If you really think that you need this filter, please use a custom method name instead of overriding the super method.
```suggestion
    def get_flagged_comments(self):
```

Reply:
> ```python
> Comment.objects.all()  # => the result is all comments are FLAGGED
> ```

This function is just filtering out the comments that have a flag count less than a certain number(Currently, the logic is flawed as it returns the reverse i.e. comments that have flag count greater than the number.). I think I might be missing something here as I don't understand your concern.

Reply:
My concern is to NOT overriding the manager method `get_queryset`
Please read this [modifying a manager's initial queryset](https://docs.djangoproject.com/en/3.0/topics/db/managers/#modifying-a-manager-s-initial-queryset)

Reply:
I was overriding this method so that all operations related to comments should only be allowed on instances that have not crossed the allowed flag count criteria. 

These comments should no longer be displayed on a `post`? Am I still missing something?

Reply:
No. I was actually missing something 😃. So you think we should hide the comments that exceed the allowed falgs? why don't we let the developer decide whether hiding them or not??

Reply:
If they want to do it, they can set the count to 0.